app/javascript/dashboard/routes/dashboard/settings/attributes/EditAttribute.vue

```diff
@@ -0,0 +1,205 @@
+              : ''
+          "
+          :placeholder="$t('ATTRIBUTES_MGMT.ADD.FORM.NAME.PLACEHOLDER')"
+          @blur="$v.displayName.$touch"
+        />
+        <label :class="{ error: $v.description.$error }">
+          {{ $t('ATTRIBUTES_MGMT.ADD.FORM.DESC.LABEL') }}
+          <textarea
+            v-model.trim="description"
+            rows="5"
+            type="text"
+            :placeholder="$t('ATTRIBUTES_MGMT.ADD.FORM.DESC.PLACEHOLDER')"
+            @blur="$v.description.$touch"
+          />
+          <span v-if="$v.description.$error" class="message">
+            {{ $t('ATTRIBUTES_MGMT.ADD.FORM.DESC.ERROR') }}
+          </span>
+        </label>
+        <label :class="{ error: $v.attributeModel.$error }">
+          {{ $t('ATTRIBUTES_MGMT.ADD.FORM.MODEL.LABEL') }}
+          <select v-model="attributeModel">
+            <option v-for="model in models" :key="model.id" :value="model.id">
+              {{ model.option }}
+            </option>
+          </select>
+          <span v-if="$v.attributeModel.$error" class="message">
+            {{ $t('ATTRIBUTES_MGMT.ADD.FORM.MODEL.ERROR') }}
+          </span>
+        </label>
+
+        <label :class="{ error: $v.attributeType.$error }">
+          {{ $t('ATTRIBUTES_MGMT.ADD.FORM.TYPE.LABEL') }}
+          <select v-model="attributeType">
+            <option v-for="type in types" :key="type.id" :value="type.id">
+              {{ type.option }}
+            </option>
+          </select>
+          <span v-if="$v.attributeType.$error" class="message">
+            {{ $t('ATTRIBUTES_MGMT.ADD.FORM.TYPE.ERROR') }}
+          </span>
+        </label>
+        <div v-if="displayName" class="medium-12 columns">
+          <label>
+            {{ $t('ATTRIBUTES_MGMT.ADD.FORM.KEY.LABEL') }}
+            <i class="ion-help" />
+          </label>
+          <p class="key-value text-truncate">
+            {{ attributeKey }}
+          </p>
+        </div>
+      </div>
+      <div class="modal-footer">
+        <woot-button :is-loading="uiFlags.isCreating">
+          {{ $t('ATTRIBUTES_MGMT.EDIT.UPDATE_BUTTON_TEXT') }}
+        </woot-button>
+        <woot-button variant="clear" @click.prevent="onClose">
+          {{ $t('ATTRIBUTES_MGMT.ADD.CANCEL_BUTTON_TEXT') }}
+        </woot-button>
+      </div>
+    </form>
+  </div>
+</template>
+
+<script>
+import { mapGetters } from 'vuex';
+import { required, minLength } from 'vuelidate/lib/validators';
+import alertMixin from 'shared/mixins/alertMixin';
+export default {
+  components: {},
+  mixins: [alertMixin],
+  props: {
+    selectedAttribute: {
+      type: Object,
+      default: () => {},
+    },
+  },
+  data() {
+    return {
+      displayName: '',
+      description: '',
+      attributeModel: 0,
+      attributeType: 0,
+      models: [
+        {
+          id: 0,
+          option: 'Conversation',
+        },
+        {
+          id: 1,
+          option: 'Contact',
+        },
+      ],
+      types: [
```

Comment:
The constants `models` and `types` are used in both AddAtrribute and EditAttribute components. Let's move this into constants.